marketplace-service/src/main/java/com/axonivy/market/service/impl/ProductServiceImpl.java

```diff
@@ -216,37 +223,37 @@ private void updateLatestChangeToProductsFromGithubRepo() {
   private void modifyProductLogo(String parentPath, GitHubFile file, Product product, GHContent fileContent) {
     Product result;
     switch (file.getStatus()) {
-    case MODIFIED, ADDED:
-      var searchCriteria = new ProductSearchCriteria();
-      searchCriteria.setKeyword(parentPath);
-      searchCriteria.setFields(List.of(MARKET_DIRECTORY));
-      result = productRepository.findByCriteria(searchCriteria);
-      if (result != null) {
-        result.setLogoUrl(GitHubUtils.getDownloadUrl(fileContent));
-        productRepository.save(result);
-      }
-      break;
-    case REMOVED:
-      result = productRepository.findByLogoUrl(product.getLogoUrl());
-      if (result != null) {
-        productRepository.deleteById(result.getId());
-      }
-      break;
-    default:
-      break;
+      case MODIFIED, ADDED:
+        var searchCriteria = new ProductSearchCriteria();
+        searchCriteria.setKeyword(parentPath);
+        searchCriteria.setFields(List.of(MARKET_DIRECTORY));
+        result = productRepository.findByCriteria(searchCriteria);
+        if (result != null) {
+          result.setLogoUrl(GitHubUtils.getDownloadUrl(fileContent));
+          productRepository.save(result);
+        }
+        break;
+      case REMOVED:
+        result = productRepository.findByLogoUrl(product.getLogoUrl());
+        if (result != null) {
+          productRepository.deleteById(result.getId());
+        }
+        break;
+      default:
+        break;
     }
   }
 
   private void modifyProductByMetaContent(GitHubFile file, Product product) {
     switch (file.getStatus()) {
-    case MODIFIED, ADDED:
-      productRepository.save(product);
-      break;
-    case REMOVED:
-      productRepository.deleteById(product.getId());
-      break;
-    default:
-      break;
+      case MODIFIED, ADDED:
```

Comment:
Did you change any logic from 249 to 256? If no, please revert the change for that block 249 to 256